test/jdk/java/awt/print/PrinterJob/PrintNullString.java

```diff
@@ -169,7 +169,7 @@ private void paint(Graphics2D g2d) {
             }
 
             try {
-                g2d.drawString(emptyIterator, 20, 180);
+                g2d.drawString(emptyIterator, 20.0f, 180.0f);
```

Comment:
Does the line below also need to change ?
`g2d.drawString("FAILURE: No IAE for empty iterator, float", 20.0f 180.0f);`

Reply:
No, it does not. It may though.

The test declares it verifies `drawString(Iterator, float, float)` — the test does it now. How the error message is output on the screen isn't important: either `int` or `float` version will do the same thing.

Reply:
@aivanov-jdk Thank you for clarification.